### PR TITLE
feat: 139 post /users/userid/UI config

### LINKED ISSUE #139: POST /users/[userid]/uiConfig
subtask of 
- [x] https://github.com/ospin-web-dev/hambda/issues/2178

blocked by 
- [x]  https://github.com/ospin-web-dev/hambda/issues/2231

### PR DESCRIPTION
**Pull request by cairesana**
- closes #139

### CODE REVIEW

File: test/src/uIConfig/userUIConfig/post.test.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+const { DEFAULT_REQ_OPTS } = require('../../../../src/utils/defaultReqOpts')
+
+describe('post', () => {
+  afterAll(() => { jest.restoreAllMocks() })
+
+  const params = {
+    userId: faker.datatype.uuid(),
+  }
+
+  it('calls amplify\'s API.get method', async () => {
```
Comment (lev-leontyev-ospin) [inline test/src/uIConfig/userUIConfig/post.test.js]: name doesn't match the test itself

**Review by lev-leontyev-ospin — DISMISSED** [review]

**Comment by lev-leontyev-ospin** [comment]
One non-blocking point.

**Comment by cairesana** [comment]
@Lev, I have updated the name, thanks for catching this

File: src/uIConfig/userUIConfig/post.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+const { DEFAULT_REQ_OPTS } = require('../../utils/defaultReqOpts')
+
+/**
+ * @desc returns a UI config for a user
+ * @memberof nexus.uIConfig.userUIConfig
+ * @function post
+ * @async
+ * @param {object} params
+ * @param {string} params.userId
+ * @returns {Promise<ApiResponse>}
```
Comment (FelixMZ2018) [inline src/uIConfig/userUIConfig/post.js]: this definition would mean the function is called like this

```
fn ({userId}) 
```

but thats not what we want but instead we have 2 params , the first beeing a string and the other one beeing an object (but we dont define the keys of the object here( so no params.something, simple params would be enough) 

```
 * @param {object} params
 * @param {string} params.userId
 ```
is a single function param, 

```
 * @param {object} params
 * @param {string} params2
 ```
 means the function has 2 params

**Review by FelixMZ2018 — CHANGES_REQUESTED** [review]
there is a small mistake in the docs

File: src/uIConfig/userUIConfig/post.js (hunk 2 above)
Comment (cairesana) [reply src/uIConfig/userUIConfig/post.js]: done

**Review by FelixMZ2018 — DISMISSED** [review]

**Review by vdiezel — APPROVED** [review]

**Pull request merged**